codeToRunTable1.R

```diff
@@ -1,14 +1,22 @@
+# if you want to run this code, you need to have the following packages installed:
+# install.packages("remotes")
+# remotes::install_github("OHDSI/DatabaseConnector@v6.2.3")
+# remotes::install_github("OHDSI/PatientLevelPrediction@v6.3.5")
+# remotes::install_github("OHDSI/CohortGenerator@v0.8.1")
+# install.packages("stringr")
+
 
 library(remotes)
 library(stringr)
-library(DatabaseConnector)
-library(PatientLevelPrediction)
-library(CohortGenerator)
+library(DatabaseConnector) # remotes::install_github("OHDSI/DatabaseConnector@v6.2.3")
+library(PatientLevelPrediction) # remotes::install_github("OHDSI/PatientLevelPrediction@v6.3.5")
+library(CohortGenerator) # remotes::install_github("OHDSI/CohortGenerator@v0.8.1")
 library(CirceR)
```

Comment:
Looks like CirceR is only a suggest in CohortGenerator, so maybe also add the remotes::install_github here. This is needed to get the cohorts from json.